internal/server/server_test.go

```diff
@@ -318,8 +318,8 @@ func TestComposeImage(t *testing.T) {
 		}
 		response, body := tutils.PostResponseBody(t, "http://localhost:8086/api/image-builder/v1/compose", payload)
 		require.Equal(t, 400, response.StatusCode)
-		require.Contains(t, body, "image_requests/0/upload_request/options")
-		require.Contains(t, body, "Value is not nullable")
+		require.Regexp(t, "image_requests/0/upload_request/options|image_requests/0/upload_request/type", body)
```

Comment:
It seems that the order in which openapi3filter checks for invalid input is not strict - I've seen both of these errors regularly and both are equally technically correct, hence this change from require.Contains to require.Regexp in order to catch both.